packages/dd-trace/src/encode/0.4.js

```diff
@@ -195,8 +195,6 @@ class AgentEncoder {
     let length = 0
 
     for (const key of keys) {
-      if (typeof value[key] !== 'string' && typeof value[key] !== 'number') return
```

Comment:
we're now encoding dictionaries within dictionaries so we need to remove this

Reply:
Do we? I thought these were converted to strings?

Reply:
this `_encodeMap` function is encoding the whole payload: 
```javascript
    const payload = {
      version: ENCODING_VERSION,
      metadata: {
        'language': 'javascript',
        'library_version': tracerVersion,
        'runtime.name': 'node',
        'runtime.version': process.version
      },
      events: this._events.map(formatSpan)
    }
```

Reply:
done in https://github.com/DataDog/dd-trace-js/pull/1880/commits/4b6286423f62b5d0343bd9ecc81a9f243a8191fc